### PR TITLE
feat(content): Non-bribable Pug planets

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug fix**

This PR addresses the bug/feature described in issue #10209.

## Summary
The Pug planets are no longer able to be bribed until after switching to Neutral. Otherwise, it is possible to completely destroy hundreds of Pug ships and then land on their planets. Although the Republic has the same problem, the Pug are definitely more proud and would like it a lot less. 

## Screenshots
No

## Usage examples
N/A

## Testing Done
None

## Save File
This save file can be used to test these changes:
None. You can start a Forge or Omnis save. 

## Artwork Checklist
N/A

## Wiki Update
N/A

## Performance Impact
N/A

## Alternative Approaches
Make the Pug planets unable to be bribed _only after_ the war with them starts.

### CODE REVIEW

**Review by ziproot — CHANGES_REQUESTED** [review]
If you want to fix #10209, you should make “Deneb” have bribe 0, not the Pug government, which already has bribe 0 by default.

**Review by ziproot — APPROVED** [review]
This now fixes #10209

**Review by ziproot — CHANGES_REQUESTED** [review]
Oops, my bad, this also needs to be done for “fwc pug defeated”

File: data/human/campaign events.txt [hunk 1]
```diff
@@ -3153,8 +3155,10 @@ event "fwc pug defeated"
 		government "Republic"
 	planet "Pugglemug"
 		remove attributes
+		"bribe" 0
```
Comment (ziproot) [inline data/human/campaign events.txt:3158]:
```suggestion
		"bribe" 0.01
```

File: data/human/campaign events.txt [hunk 2]
```diff
@@ -3153,8 +3155,10 @@ event "fwc pug defeated"
...
 	planet "Pugglequat"
 		remove attributes
+		"bribe" 0
```
Comment (ziproot) [inline data/human/campaign events.txt:3161]:
```suggestion
		"bribe" 0.01
```

**Review by ziproot — CHANGES_REQUESTED** [review]

**Review by ziproot — APPROVED** [review]
There we go

**Comment by TheGiraffe3** [comment]
Come to think of it, should I add this to the planet in Pug Iyik as well?

**Comment by ziproot** [comment]
> Come to think of it, should I add this to the planet in Pug Iyik as well?

Vera Pug already cannot be bribed.

**Comment by TheGiraffe3** [comment]
Oh…

**Comment by TheGiraffe3** [comment]
I think this is ready for review/merge.

**Comment by TheGiraffe3** [comment]
This is ready for review/merge.

**Comment by warp-core** [comment]
I'm not sure pride is a reason for the Pug to not accept your bribe to land.
I'm not even sure proud is an appropriate description of them here. They know humans can defeat the ships they operate here: that's actually the point. The Pug as a whole are still far superior, regardless of what the player does here.

**Comment by Saugia** [comment]
I agree; I don't think it's a correct assumption that the Pug are "prideful". If anything, it makes much more sense that they're the opposite of the Quarg - the Quarg would acknowledge you and your behavior as a human, especially when you attack innocents or especially their own ships. Meanwhile, it would make sense that the Pug would not care whatever you do as a human. They would brush you off; never see you as a threat. The galaxy is a playground to them - they probably wouldn't care if you broke even thousands of their toys.

**Comment by TheGiraffe3** [comment]
Okay, "proud" is the wrong word. But the Pug aren't wanting to get destroyed _before_ the war starts.
Also, the Vara Pug Pug don't let you bribe them.

**Pull request merged**